MapboxNavigation/NavigationViewController.swift

```diff
@@ -618,7 +618,7 @@ extension NavigationViewController: RouteControllerDelegate {
         self.delegate?.navigationViewController?(self, willRerouteFrom: location)
     }
     
-    @objc public func routeController(_ routeController: RouteController, didRerouteAlong route: Route) {
```

Comment:
I think nested types like this aren't actually accessible to objc. 

I can move the type, but I wanted to be able to test it. I was wondering if anyone had advice for quickly testing the objc build of the library, or if I should just bit the bullet and set up a purpose built objc project.

Reply:
To be honest, I have no experience with objc. So I can't really help here.

Reply:
Given that this existed previously, I would be fine with just leaving it as it is right now until someone has an actual issue and we can always fix this later?

Reply:
I've done some objc and swift development, but I'm new to SPM. I haven't tried too hard, but I wasn't immediately able to figure out how to use this library from objc after installing from spm.


It's fine by me to leave it as is for now